src/analysis/completion.ml

```diff
@@ -185,7 +185,9 @@ let make_candidate ~get_doc ~attrs ~exact ~prefix_path name ?loc ?path ty =
     | `Mod m   ->
       begin try
           if not exact then raise Exit;
-          let verbosity = !Type_utils.verbosity in
+          let verbosity = 
+            Mconfig.Verbosity.to_int !Type_utils.verbosity ~for_smart:1
```

Comment:
In most cases, I tried to keep in mind ocaml-lsp's use case for `verbosity=smart`, but I'm not sure what to put here.